pkg/utils/json.go

```diff
@@ -42,3 +47,51 @@ func removeNaN(hosts []result.Host) []result.Host {
 	}
 	return newHosts
 }
+
+func GetMetics(searchDir string, m *[]metrics.Metrics) error {
+	resultFilePath := path.Join(path.Dir(path.Clean(searchDir)), "result.txt")
+
+	bytes, err := ioutil.ReadFile(resultFilePath)
+	if err != nil {
+		return err
+	}
+
+	r, err := regexp.Compile(`{[^}]*}`)
```

Comment:
What is this regex doing?

Reply:
To parse the any string that looks like a json.

Reply:
Done. I need to understand the google results instead of is-works. ^_^